R/finalize_plot.R

```diff
@@ -315,29 +338,113 @@ finalize_plot <- function(input_plot = NULL,
                     col = cmapplot_globals$colors$blackish)
   )
 
+
+  # Function to create plot object with left aligned legend on top
+  buildChart <- function(p,l_margins,p_margins,l_indent,l_height,p_height) {
```

Comment:
This function should be outside of finalize_plot() so that it gets built once by the package, as opposed to recreated each time someone runs finalize. In terms of documentation, see how Noel documented the ggplot_size_conversion function over in cmapplot.R.

Reply:
Done